.github/steps/2-tbd.md

```diff
@@ -21,64 +21,65 @@ This page contains the list of secret scanning alerts. You can filter and sort t
 
 - **Amazon AWS Secret Access Key**: This is the access key you committed in the last step
 - **Amazon AWS Access Key ID**: This is the key ID committed in the last step
-- **GitHub Personal Access Token**: This token was already in the `credentials.yml`before you got started
+- **GitHub Personal Access Token**: This token was already in the `credentials.yml` file before you started
 
 ### :keyboard: Activity 2.2: Review a secret scanning alert
 
-In this activity, you will explore the alert UI. You'll review the validity of the secret and identify where the secret was detected in the repository.
+In this activity, you will explore the alert. You'll review the validity of a secret and identify where the secret was detected in the repository.
 
-Open the **Amazon AWS Access Key ID** alert.
+Open the **Amazon AWS Access Key ID** alert and explore the information shown.
 
-**Alert status:** This section identifies the current status of the alert (open or closed) and identifies when the alert was first detected.
+- **Alert status:** This section shows the current status of the alert (open or closed) and reports when the alert was first detected.
 
-![A screenshot of the Amazon AWS Access Key ID alert with the currently open status highlighted](/images/alert-status.png)
+  ![Screenshot of the Amazon AWS Access Key ID alert with the currently open status highlighted.](/images/alert-status.png)
```

Comment:
I indented this section of information to make it easier to see the instruction. I missed this when I scanned the page because it's unnumbered, which makes sense since it's the only step in this section.